Code/Tools/ProjectManager/Source/ProjectsScreen.cpp

```diff
@@ -333,7 +336,21 @@ namespace O3DE::ProjectManager
                 }
             }
 
-            // Let the user can cancel builds for projects in the build queue
+            if (m_currentExporter)
+            {
+                AZ::IO::Path exportProjectPath = AZ::IO::Path(m_currentExporter->GetProjectInfo().m_path.toUtf8().constData());
+                if (!exportProjectPath.empty())
+                {
+                    //Setup export button
+                    auto exportProjectIter = m_projectButtons.find(exportProjectPath);
+                    if (exportProjectIter != m_projectButtons.end())
```

Comment:
nit: you condense this to a single statement
```suggestion
                    if (auto exportProjectIter = m_projectButtons.find(exportProjectPath); exportProjectIter != m_projectButtons.end())
```